### ISSUE TITLE
Run multiple instances of helmsman

### ISSUE THREAD

**Issue by galindro** [Praqma/helmsman#529]
I use Helmsman to deploy the same DSF to multiple clusters. In my CI/CD pipeline, there is a stage that runs helmsman for all clusters at the same time. The problem is that all of them uses the same build executor and thus, the same filesystem which contains the dsf file. I think that, every time that helmsman runs, it deletes the .helmsman-tmp/ directory, causing trouble to the other helmsman executions. Is there a way to solve this or do I need to use different build executors?

**Comment by mkubaczyk** [comment]
As helmsman creates its tmp dir based on where it is being run, you should be able to run it in parallel (multiple helmsman's run) just by having each DSF in its dedicated directory. I know it's not the prettiest, but should work. You can even keep the DSF together in repo, just do this "magic" on CI level

**Comment by antoinedeschenes** [comment]
tmpdir could easily be randomized, but you'll also find that the `helm repo update` command run by helmsman causes issues when running in parallel (setting HELM_ environment variables could help there)

**Comment by epierotto** [comment]
Another issue that you may find is while working with multiple KUBECONFIG or `kubectl` contexts.
Helmsman runs `kubectl config use-context my-context` instead of ensuring that each `kubectl` or `helm` command has the flags `--kubeconfig` and `--context`
If you are running multiple Helmsman at the same time this is a big issue.
